superbench/runner/runner.py

```diff
@@ -58,6 +58,27 @@ def __get_enabled_benchmarks(self):
                 return list(self._sb_config.superbench.enable)
         return [k for k, v in self._sb_benchmarks.items() if v.enable]
 
+    def __get_mode_command(self, mode, exec_command):
+        """Get runner command for given mode.
+
+        Args:
+            mode (str): Runner mode.
+            exec_command (str): Executor command.
+
+        Return:
+            str: Runner command.
+        """
+        if mode == 'torch.distributed':
```

Comment:
How to distinguish "distributed training" and "local parallel training per single node"?

Reply:
there's no need to distinguish? the only difference is `nnode=1` or `nnode>1`

Reply:
added `proc_num` and `node_num` config